examples/kubernetes/dynamic-provisioning/filesystem/storageclass.yaml

```diff
@@ -0,0 +1,28 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: fsx-sc
+provisioner: fsx.openzfs.csi.aws.com
+parameters:
+  volumeType: "filesystem" #REQUIRED
+  kmsKey: "12345678-90ab-cdef-ghij-klmnopqrstuv"
+  automaticBackupRetentionDays: "1"
+  copyTagsToBackups: "false"
+  copyTagsToVolumes: "false"
+  dailyAutomaticBackupStartTime: "00:00"
+  deploymentType: "SINGLE_AZ_1" #REQUIRED
+  diskIopsConfiguration: "Iops=300,Mode=USER_PROVISIONED"
+  rootVolumeConfiguration: "CopyTagsToSnapshots=true,DataCompressionType=NONE,NfsExports=[{ClientConfigurations=[{Clients=*,Options=[rw,crossmnt]}]}],ReadOnly=false,RecordSizeKiB=128,UserAndGroupQuotas=[{Id=1,StorageCapacityQuotaGiB=10,Type=USER}]"
+  throughputCapacity: "64" #REQUIRED
+  weeklyMaintenanceStartTime: "7:09:00"
+  securityGroupIds: "sg-068000ccf82dfba88"
+  subnetIds: "subnet-0eabfaa81fb22bcaf" #REQUIRED
+  tags: "Tag1=Value1,Tag2=Value2"
+  skipFinalBackup: "true" #REQUIRED
+reclaimPolicy: Delete
+allowVolumeExpansion: true
+mountOptions:
+  - nfsvers=4.1
+  - rsize=10408576
```

Comment:
The maximum nfs ```rsize``` and ```wsize``` for a 64 MB/s throughput FSxZ file system is actually ```262144```. Entering a larger value will default to ```262144```: https://docs.aws.amazon.com/fsx/latest/OpenZFSGuide/other-mount-considerations.html. I had these values wrong in my design, but it's probably better to put the valid maximum value here.

Reply:
Maybe we should adjust it to the normal default of `1048576` instead. In the example I wanted to keep the capacity and throughput at the minimums to reduce cost. With that said users will most likely increase these values but may possibly forget to update the `rsize` and `wsize`. Since there's no harm in setting these to the max, we can help ensure users experience max performance.